tests/planning_poker/test_consumers.py

```diff
@@ -1,22 +1,40 @@
 from unittest.mock import Mock, patch
 
+from channels.testing import WebsocketCommunicator
 import pytest
 
+from example.asgi import application
+
 
 class TestPokerConsumer:
     def test_poker_session(self, poker_consumer):
         assert poker_consumer.poker_session.id == 1
 
-    # @pytest.mark.django_db
-    # @pytest.mark.asyncio
-    # async def test_connect(self):
-    #     application = URLRouter([
-    #         url(r'/(?P<poker_session>\w+)/$', PokerConsumer)
-    #     ])
-    #     communicator = WebsocketCommunicator(application, 'ws://test/planning_poker/1/')
-    #     connected, subprotocol = await communicator.connect()
-    #     assert connected
-    #     await communicator.disconnect()
+    HAS_NO_ACTIVE_STORY = 1
+    HAS_ACTIVE_STORY = 2
+
+    # The table is prepopulated with two poker sessions. The first one has no active story, and the second has one.
+    @pytest.mark.parametrize('poker_session_id', [HAS_NO_ACTIVE_STORY, HAS_ACTIVE_STORY])
+    @pytest.mark.asyncio
+    @pytest.mark.django_db
+    async def test_connect(self, poker_session_id):
+        communicator = WebsocketCommunicator(application, 'ws://test/planning_poker/{}/'.format(poker_session_id))
+        connected, subprotocol = await communicator.connect()
+        assert connected
+        if poker_session_id == self.HAS_ACTIVE_STORY:
```

Comment:
Aside from the connection assertion above, this `if` means that *nothing* is really tested for the case without an active story. If no story `story_changed` event should be received in this case, then the test should explicitly check for that.